client/apps/shipping-label/state/reducer.js

```diff
@@ -208,6 +208,7 @@ reducers[ UPDATE_PACKAGE_WEIGHT ] = ( state, { packageId, value } ) => {
 	newPackages[ packageId ] = {
 		...newPackages[ packageId ],
 		weight: parseFloat( value ),
+		is_user_specified_weight: true,
```

Comment:
AFAIK the ones we have in snake case are initialized from data that originates on the PHP side of the house. (Where we aren't doing anything to data we're getting directly from WC or the WP database before passing it to `wp_localize_script()`.